lib/rspec/support/differ.rb

```diff
@@ -2,11 +2,14 @@
 RSpec::Support.require_rspec_support 'hunk_generator'
 
 require 'pp'
+require 'rspec/support/object_inspector'
 
 module RSpec
   module Support
     # rubocop:disable ClassLength
     class Differ
+      include RSpec::Support::ObjectInspector
```

Comment:
I don't think there's any reason to include this module.  It's not meant to be used as a mixin (and in fact defines no instance methods).